lib/template-mixins.js

```diff
@@ -524,12 +536,15 @@ module.exports = function (options, deprecated) {
                         throw new Error('Could not find field: ' + key);
                     }
                 }
+                if (this.disableRender) {
+                    return null;
+                }
                 if (this.html) {
                     return this.html;
                 }
                 var mixin = this.mixin || 'input-text';
                 if (mixin && res.locals[mixin] && typeof res.locals[mixin] === 'function') {
-                    return res.locals[mixin]().call(Object.assign({}, res.locals, this), this.key);
+                    return res.locals[mixin]().call(Object.assign({}, res.locals), this.key);
```

Comment:
I think that makes sense. I've cloned this and run it against some examples of calling `renderField` with a key, and it fails, but for unrelated reasons. So I'll go and fix them...